.circleci/config.yml

```
@@ -17,6 +17,19 @@ workflows:
           matrix: &go-version-matrix
             parameters:
               go_version: ["1.14.7", "1.15.15"]
+  "circleci_build_and_test_nightly":
+    triggers:
+      - schedule:
+          cron: "37 3 * * *"
+          filters:
+            branches:
+              only: compare2algod
```

Is circleCI already running this cron job out of your branch?
```suggestion
              only: develop
```

---

[yep](https://app.circleci.com/pipelines/github/algorand/indexer/519/workflows/48626736-7ea8-47fe-89c7-922fcd8e2339)

---

I am planning to point the branch to `develop` as the final step before merging the PR